packages/core/admin/admin/src/features/Auth.tsx

```diff
@@ -231,7 +231,10 @@ const AuthProvider = ({
       const matchingPermissions = actualUserPermissions.filter(
         (permission) =>
           permissions.findIndex(
-            (perm) => perm.action === permission.action && perm.subject === permission.subject
+            (perm) =>
+              perm.action === permission.action &&
+              // Only check the subject if it's provided
+              (perm.subject == undefined || permission.subject === permission.subject)
```

Comment:
Shouldn't this be:

```suggestion
              (perm.subject == undefined || perm.subject === permission.subject)
```

If so I wonder if the undefined check is still necessary 🤔

Reply:
Oof that would have been bad, thanks for spotting it